### PR TITLE
[10.0][mig][crm_meeting_commercial_partner]

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Migration from https://github.com/OCA/crm/pull/208

CRM Meeting Commercial Partner
==============================

This module allows to display the meetings of a company as the sum of all
the meetings held with the company's contacts.

Usage
=====
Go to the partner views. You will see, in the Kanban and Form views, that
the number of meetings shown for a company includes the meetings held with
the company's contacts.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
What is the advantage of this over checking the "All" calendar?

**Comment by JordiBForgeFlow** [comment]
@pedrobaeza When you go to a partner, say "Eficent", you want to see how many meetings are scheduled with Eficent's individual persons, because as a general rule no one meets with Eficent as a company.

**Comment by pedrobaeza** [comment]
OK, got it. I was thinking on the company, not on the partner side. But this is not correctly put nor named. It should be `calendar_commercial_partner` or `calendar_commercial_partner_meeting`, and be relocated to OCA/calendar, not here.

**Comment by JordiBForgeFlow** [comment]
But this feature depends on module `crm`, not `calendar.` That's why I suggested it here

**Comment by pedrobaeza** [comment]
I think you should go more for the generic one (these CRM meetings are also calendar meetings).

**Comment by JordiBForgeFlow** [comment]
@pedrobaeza This module builds on top of a feature to display the partner's meetings that is introduced by odoo in the CRM module.
https://github.com/odoo/odoo/blob/10.0/addons/crm/models/res_partner.py#L37
https://github.com/odoo/odoo/blob/10.0/addons/crm/views/res_partner_views.xml#L119

It's all over the crm module.

What this module does is to extend the methods to allow the user to view all the meetings from a company perspective, to have an overall view of all meetings scheduled with individual contacts.

By removing the dependency with crm we would duplicate the features.

**Comment by pedrobaeza** [comment]
Ok, let's go this way.

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Comment by JordiBForgeFlow** [comment]
@bjeficent @ageficent can you test this on runbot? @lreficent can help you.

File: crm_partner_schedule_meeting/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -*- encoding: utf-8 -*-
```
Comment (AdriaGForgeFlow) [inline crm_partner_schedule_meeting/__manifest__.py]: I think this file does not belong here. It makes Odoo crash. Solved deleting the folder.

**Review by AdriaGForgeFlow — CHANGES_REQUESTED** [review]
Odoo crashing on startup due to crm_partner_shcedule_meeting/ folder.

**Comment by LoisRForgeFlow** [comment]
@ageficent It's fixed, could you update your review?

**Review by AdriaGForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**